VstsTasks/AzureDtlCreateVM/task.json

```diff
@@ -12,7 +12,7 @@
     "version": {
         "Major": 2,
         "Minor": 0,
-        "Patch": 30
+        "Patch": 31
```

Comment:
doesn't it need to match with the version mentioned in vss-extension.json. if not, why are there 2 different versions.

Reply:
The extension version is for the overall set of tasks. The task version is specific. Extension version needs to be incremented or it will not be updated in the Marketplace. The task version needs to be updated or the new scripts will not be used.